integreat_cms/cms/views/bulk_action_views.py

```diff
@@ -162,12 +162,26 @@ def post(
             )
             return super().post(request, *args, **kwargs)
 
-        for content_object in self.get_queryset():
-            if not content_object in to_translate:
+        already_up_to_date_translations = [
+            content_object.best_translation.title
+            for content_object in self.get_queryset()
+            if content_object not in to_translate
+        ]
+        match already_up_to_date_translations:
+            case []:
+                pass
+            case [single]:
                 messages.error(
                     request,
                     _("There already is an up-to-date translation for {}").format(
-                        content_object.best_translation.title,
+                        single
+                    ),
+                )
+            case _:
+                messages.error(
+                    request,
+                    _("There already are up-to-date translations for {}").format(
+                        iter_to_string(already_up_to_date_translations)
                     ),
                 )
```

Comment:
Since the message only differs on pluralization, I think this would be a great use case for `ngettext_lazy()` (has to be imported from `django.utils.translation`):

```
        if already_up_to_date_translations:
            messages.error(
                request,
                ngettext_lazy(
                    "There already is an up-to-date translation for {}.",
                    "There already are up-to-date translations for {}.",
                    len(already_up_to_date_translations),
                ).format(iter_to_string(already_up_to_date_translations)),
            )
```